Docs review of MDA private preview Deploy (#4336)
langchain-ai/docs: src/langsmith/managed-deep-agents-deploy.mdx

Reviewer: ```suggestion
To give a subagent its own MCP tools, add `subagents/researcher/tools.json` with the same shape as the project-level `tools.json`. Subagent names are checked case insensitively for duplicates.
```

description: Create or update a Managed Deep Agent with the CLI or REST API.
---

> LangChain hosts Managed Deep Agents, so you can deploy an agent without running your own infrastructure. Deploying creates or updates the hosted agent resource and syncs the managed file tree of instructions, skills, subagents, and tool configuration. A deploy does not create a [LangSmith Deployment](/langsmith/deployment-quickstart). See [Created resources](/langsmith/managed-deep-agents-overview#created-resources) for what it does create.

Reviewer: ```suggestion
LangChain hosts Managed Deep Agents, so you can deploy an agent without running your own infrastructure. Deploying creates or updates the hosted agent resource and syncs the managed file tree of instructions, skills, subagents, and tool configuration. A deploy does not create a [LangSmith Deployment](/langsmith/deployment-quickstart). For more information, on what a deploy does create, refer to [Created resources](/langsmith/managed-deep-agents-overview#created-resources).
```

Use the [CLI](/langsmith/managed-deep-agents-cli) for most setups. Use the [REST API](/langsmith/managed-deep-agents-api-overview) when you need a custom client, cannot run the CLI from your automation, or want direct control over request payloads.

<Note>
Managed Deep Agents is in **private preview**, available on [LangSmith Cloud](/langsmith/cloud) in the US region only. [Join the waitlist](https://www.langchain.com/langsmith-managed-deep-agents-waitlist) to request access.
</Note>

> For a faster path, see the [quickstart](/langsmith/managed-deep-agents-quickstart).

Reviewer: Instead of just "faster path" should we be more descriptive of why you would use this page over the quickstart. Like maybe "This page provides setup details for x, y, z, for a faster setup, refer to the quickstart..."

cd my-agent
```

The command generates:

> | File or directory | Purpose |

Reviewer: is this the same as the table on the other mda page? Just wondering if a snippet could be useful... maybe not 🤔

  subagents/<name>/tools.json      # Optional: subagent-scoped MCP tools
```

> `deepagents init` generates an empty `tools.json`, one example skill, and one example subagent so the initial deploy succeeds before you register an MCP server. Edit or remove the examples to fit your agent. Deploy reads every file in the project and syncs the tree to the Context Hub agent repo. For the complete field reference, see the [CLI reference](/langsmith/managed-deep-agents-cli#project-file-reference).

Reviewer: do you think it's worth linking the relevant pages/section for skill, subagent,  Context Hub?

| Update an agent | [`PATCH /v1/deepagents/agents/{agent_id}`](/langsmith/managed-deep-agents-api/agents/update-agent) |
| Delete an agent | [`DELETE /v1/deepagents/agents/{agent_id}`](/langsmith/managed-deep-agents-api/agents/delete-agent) |

> The create route, `POST /v1/deepagents/agents`, is shown above under [Create an agent](#create-an-agent).

Reviewer: ```suggestion
For an example of the create route, `POST /v1/deepagents/agents`, refer to [Create an agent](#create-an-agent).
```
